CMakeLists.txt

```diff
@@ -748,6 +748,16 @@ INSTALL(DIRECTORY include/ ${CMAKE_BINARY_DIR}/include/
   COMPONENT includes
   FILES_MATCHING PATTERN "*.h")
 
+# examples:
+INSTALL(DIRECTORY cookbooks/
+  DESTINATION cookbooks
+  COMPONENT examples
+  FILES_MATCHING PATTERN "*")
```

Comment:
Do we want to be more specific about what files we want to install? Do you want everything (including documentation sources, readmes, etc.)? deal.II uses patterns here.